molecule_requirements.txt

```diff
@@ -0,0 +1,2 @@
+jmespath
+git+https://github.com/tyll/selinux-pypi-shim@fulllocation
```

Comment:
do we still need this, when https://github.com/pycontribs/selinux/pull/3 is already merged and released?

Reply:
Original commit: https://github.com/linux-system-roles/network/pull/107/commits/f9fff1776d64b37432215eee8cc30a587df808ce#diff-b91f3d5bd63fcd17221b267e851608e8

Reply:
I suppose no. Just `selinux` should work.